src/main/resources/data/minecolonies/tags/items/crafting_ignores_nbt.json

```diff
@@ -0,0 +1,10 @@
+{
+  "replace": false,
+  "values": [
+    "#minecraft:banners",
+    "minecraft:clock",
+    "minecraft:compass",
+    "minecolonies:ancienttome",
+    "minecolonies:colony_banner"
+  ]

```

Comment:
Can we add the vanilla books you can write inside to this too?

Reply:
I'm a bit reluctant to do that, as (unless I'm missing something) that would allow it to consume books the player has written important notes inside and happened to leave in their warehouse (and some players use their warehouse as primary storage, so that isn't too far-fetched).

Reply:
fair